pom.xml

```diff
@@ -852,5 +862,23 @@
                 <maven.compiler.target>${env.JAVA_VERSION}</maven.compiler.target>
             </properties>
         </profile>
+        <profile>
+            <id>jdk-8-config</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <javadocExecutable>${java.home}/../bin/javadoc</javadocExecutable>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdk-11-config</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
+            </properties>
```

Comment:
out of curiosity, what does it addition mean?

Reply:
It's described in the commit comment:

> When building with "make JAVA_VERSION=11 ..." I got an error about JAVA_HOME
not being set:
  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.0:jar
  (attach-javadocs) on project kafka-agent: MavenReportException: Error while
  generating Javadoc: Unable to find javadoc command: The environment
  variable JAVA_HOME is not correctly set. -> [Help 1]
>
> Rather than forcing people to set JAVA_HOME (which is the fallback maven-javadoc-plugin uses for finding the javadoc executable) configure the executable location directly according to the JVM being used to run maven.
>
> See https://stackoverflow.com/questions/57081473/how-to-make-maven-javadoc-plugin-work-with-any-java-version

Reply:
thanks